src/car/get.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-env serviceworker, browser */
+/* global Response */
+
+/**
+ * @typedef {import('../env').Env} Env
+ */
+
+/**
+ * Handle car get request, materializing CAR from the index.
+ *
+ * @param {Request} request
+ * @param {Env} env
+ * @param {import('..').Ctx} ctx
+ */
+export async function carGet (request, env, ctx) {
+  return new Response('Not yet implemented')
```

Comment:
Should probably have an error status code instead of 200. Also maybe it's better to remove these routes until they are implemented.